setup.py

```diff
@@ -38,7 +38,7 @@ def package_files(directory):
 
     author='ETH',
 
-    license='GNU General Public License',
+    license='OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)',
```

Comment:
This is the official PyPI string for LGPLv3: https://pypi.org/search/?q=&o=&c=License+%3A%3A+OSI+Approved+%3A%3A+GNU+Lesser+General+Public+License+v3+%28LGPLv3%29